docs/nodes/validate/add-a-validator.md

```diff
@@ -141,166 +143,94 @@ to run the example scripts directly from an AvalancheJS directory.
-  1: "avax",
-  2: "cascade",
-  3: "denali",
-  4: "everest",
-  5: "fuji",
-  1337: "custom",
-  12345: "local"
-```
-
-</p>
-</details>
 
 #### Settings for Validation
 
 Next we need to specify the node's validation period and delegation fee.
 
-```ts
-const nodeID: string = "NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg";
-const startTime: BN = UnixNow().add(new BN(60 * 1));
-const endTime: BN = startTime.add(new BN(26300000));
-const delegationFee: number = 10;
-```
-
-#### Node ID
 
-This is the node ID of the validator being added. See [above
-section](#retrieve-the-node-id) on how to retrieve the node id by using API
-[`info.getNodeID`](/reference/avalanchego/info-api.md#infogetnodeid).
+#### Validation Period
 
-#### Staking Period
+The validation period is set by default to 21 days, the start date
+being the date and time the transaction is issued. The start date
+cannot be modified.
 
-`startTime` and `endTime` are required to specify the time of starting/leaving
-validation. The minimum duration that one can validate the Primary Network is 2
-weeks, and the maximum duration is one year. One can start a new validation on
-the Primary Network after finishing one, it’s just that the maximum _continuous_
-duration is one year. `startTime` and `endTime` are the Unix times when your
-validator will start and stop validating the Primary Network, respectively.
-`startTime` must be in the future relative to the time the transaction is
-issued.
+The end date can be adjusted in the code.
 
-The sample code uses `const startTime: BN = UnixNow().add(new BN(60 * 1))` and
-`const endTime: BN = startTime.add(new BN(26300000))` to compute the Unix time 1
-minute and 304 days in the future (at the time when this article was written) to
-use as the values of `startTime` and `endTime`, respectively.
+Let's say we want the validation period to end after 50 days.
+You can achieve this by adding the number of deired days to
+`endTime.getDate()`, in this case `50`.
 
-:::tip
-
-You can create your own Unix timestamp [here](https://www.unixtimestamp.com/) or
-by using the `UnixNow()` method
-
-:::
-
-To create your own start times, please follow the steps below:
-
-Locate this line in the file
-
-```ts
-const startTime: BN = UnixNow().add(new BN(60 * 1));
-const endTime: BN = startTime.add(new BN(26300000));
+``` ts
+// move ending date 50 days into the future
+endTime.setDate(endTime.getDate() + 50);
 ```
 
-Change `startTime` and `endTime` to new `BN` values, for example:
+Now let's say you want the staking period to end on a specific 
+date and time, for example May 15, 2024, at 11:20 AM.
+It can be achieved as shown in the code below.
 
 ```ts
-const startTime: BN = new BN(1654656829); // Wed Jun 08 2022 02:53:49 GMT+0000
-const endTime: BN = new BN(1662602029); // Thu Sep 08 2022 01:53:49 GMT+0000
+const startTime = await new PVMApi().getTimestamp();
+const startDate = new Date(startTime.timestamp);
+const start = BigInt(startDate.getTime() / 1000);
```

Comment:
maybe you want to clarify that the api call requires a start argument for keeping back compatibility, but it will be ignored